providers/cncf/kubernetes/src/airflow/providers/cncf/kubernetes/operators/job.py

```diff
@@ -81,7 +81,7 @@ class KubernetesJobOperator(KubernetesPodOperator):
     :param completion_mode: CompletionMode specifies how Pod completions are tracked. It can be `NonIndexed` (default) or `Indexed`.
     :param completions: Specifies the desired number of successfully finished pods the job should be run with.
     :param manual_selector: manualSelector controls generation of pod labels and pod selectors.
-    :param parallelism: Specifies the maximum desired number of pods the job should run at any given time.
+    :param parallelism: Specifies the maximum desired number of pods the job should run at any given time. Defaults to 1
```

Comment:
See my other comments - it should be mentioned that the value should be `1` (default) or larger